meteor/client/ui/Settings/ShowStyleBaseSettings.tsx

```diff
@@ -1049,6 +1177,10 @@ const HotkeyLegendSettings = withTranslation()(
 						>
 							<th className="settings-studio-custom-config-table__name c2">{hotkeyHelper.shortcutLabel(item.key)}</th>
 							<td className="settings-studio-custom-config-table__value c3">{item.label}</td>
+							<td className="settings-studio-custom-config-table__value c2">{item.platformKey || ''}</td>
```

Comment:
Are we importing this hostKey/platformKey feature? If so, this needs to be well-documented.

Reply:
ref: https://github.com/nrkno/sofie-core/pull/694#discussion_r841458278